src/components/Focus/Focus.tsx

```diff
@@ -10,6 +11,20 @@ export interface Props {
 
 export default class Focus extends React.PureComponent<Props, never> {
   componentDidMount() {
+    this.handleSelfFocus();
+  }
+
+  componentDidUpdate(prevProps: Props) {
```

Comment:
Nitpick but you could deconstruct `{children: prevChildren, ...restPrevProps}` directly here.